### PR TITLE
extend nightly tests OS support matrix

### PR DESCRIPTION
**Pull request by meldafrawi**
https://github.com/longhorn/longhorn/issues/3085
Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@suse.com>

this PR should be merged after https://github.com/longhorn/longhorn-tests/pull/744

### CODE REVIEW

File: jenkins-jobs/master/centos/longhorn-upgrade-tests-centos-amd64.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/longhorn/longhorn-tests
+            branches:
+              - master
+      script-path: test_framework/Jenkinsfile
+      lightweight-checkout: true
+    triggers:
+      - parameterized-timer:
```
Comment (khushboo-rancher) [inline jenkins-jobs/master/centos/longhorn-upgrade-tests-centos-amd64.yml:113]: Same as above comment.

File: jenkins-jobs/master/centos/longhorn-tests-centos-amd64.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          description: "aws instance type for worker nodes"
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/longhorn/longhorn-tests
+            branches:
+              - master
+      script-path: test_framework/Jenkinsfile
+      lightweight-checkout: true
+    triggers:
```
[the remark this answers is not in the document]
Comment (innobead) [reply jenkins-jobs/master/centos/longhorn-tests-centos-amd64.yml:112]: We don’t need to run daily but on demand instead.

**Review by innobead — CHANGES_REQUESTED** [review]
Let's just keep SLES running every day, but others on demand.

P.S. we will also change our nightly build based on SLES instead of Ubuntu.

File: jenkins-jobs/master/centos/longhorn-tests-centos-amd64.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (meldafrawi) [reply jenkins-jobs/master/centos/longhorn-tests-centos-amd64.yml]: fixed, Thanks.

File: jenkins-jobs/master/centos/longhorn-upgrade-tests-centos-amd64.yml (hunk 1 above)
Comment (meldafrawi) [reply jenkins-jobs/master/centos/longhorn-upgrade-tests-centos-amd64.yml]: fixed, Thanks.

**Comment by meldafrawi** [comment]
I'll address changing default distro to SLES in another separate PR.

**Review by innobead — APPROVED** [review]

**Pull request merged**